config/locales/de/views/users.yml

```diff
@@ -0,0 +1,43 @@
+---
+de:
+  users:
+    mailer:
+      confirmation_instructions:
+        p1: 'Vielen Dank für Ihre Registrierung bei CodeHarbor! Klicken Sie auf den folgenden Link, um Ihren Account zu bestätigen:'
+      email_changed:
+        p1: Wir möchten Sie informieren, dass Ihre E-Mail-Adresse zu %{unconfirmed_email} geändert wurde.
+        p2: Wir möchten Sie informieren, dass Ihre E-Mail-Adresse zu %{email} geändert wird.
+      password_change:
+        p1: Wir möchten Sie darüber informieren, dass Ihr Passwort geändert wurde.
+      reset_password_instructions:
+        p1: Jemand hat einen Link zum Ändern Ihres Passworts angefordert. Klicken Sie auf den folgenden Link, um Ihr Passwort zu ändern.
+      unlock_instructions:
+        p1: Ihr CodeHarbor-Account wurde aufgrund einer zu großen Anzahl von erfolglosen Anmeldeversuchen gesperrt.
```

Comment:
At least those lines should be a real "duplicate", aren't they? At least I remember copying them over from the German Devise locale.

Reply:
We cannot remove them without removing the English locales too so I just went ahead and removed both.

Reply:
Alright, thanks!